cmd/registry/cmd/compute/score.go

```diff
@@ -0,0 +1,75 @@
+package compute
+
+import (
+	"context"
+
+	"github.com/apigee/registry/cmd/registry/core"
+	"github.com/apigee/registry/cmd/registry/patterns"
+	"github.com/apigee/registry/cmd/registry/scoring"
+	"github.com/apigee/registry/connection"
+	"github.com/apigee/registry/log"
+	"github.com/apigee/registry/rpc"
+	"github.com/spf13/cobra"
+)
+
+func scoreCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "score (experimental)",
+		Short: "Compute score from ScoreDefinition",
```

Comment:
Could we make this "Compute scores for APIs and API specs" to match the style of other help strings in this section?

Reply:
Done.